Sources/Sentry/Transactions/SentryTransaction.h

```diff
@@ -0,0 +1,71 @@
+//
+//  SentryTransaction.h
+//  Sentry
+//
+//  Created by Dhiogo Brustolin on 11/01/21.
+//  Copyright © 2021 Sentry. All rights reserved.
+//
+
+#import <Sentry/Sentry.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class SentrySpanContext, SentryTransactionContext, SentryHub;
+
+@interface SentryTransaction : NSObject <SentrySerializable>
```

Comment:
`h`: Please add `NS_SWIFT_NAME`

Reply:
What would the `NS_SWIFT_NAME` be? Just `Transaction`?
That's a very broad name, can we keep it SentryTransaction though?

Reply:
I will use SentryTransaction as Java does.